lib/assertions.js

```diff
@@ -690,7 +690,7 @@ module.exports = function (expect) {
         '<object> to be (a map|a hash|an object) whose values [exhaustively] satisfy <assertion>'
     ], function (expect, subject, nextArg) {
         expect.errorMode = 'nested';
-        expect(subject, 'not to equal', {});
+        expect(subject, 'not to equal', {}); // TODO: to be non-empty
```

Comment:
Thanks. And no worries. I'll remove the TODO anyway since you created the issue for it.